### PR TITLE
Move TPCH queries to auxlib, deprecate tpcc and tpch libs

### PR DESCRIPTION
**Pull request by mrzzzrm**
Moves the TPCH queries into their own files. Stores them in `const char * tpch_queries[21]". This is not a global "object" because it doesn't require initialization and is generated at compile time.

Anticipating the TPCH benchmarks, this proposes to move all TPC-H queries to a newly introduced `opossumAux` lib. Right now, this lib merges the existing `tpcc` and `tpch` libs and is intended for future code that doesn't belong into the "core" of the database but is shared among tests, console, benchmark, playgrounds etc. I prefer this single-aux-lib approach to having multiple small libraries for different purposes (tpcc, tpch, ...) because less cmake clutter.

Best timing ever with the naming changes coming up. Gonna refactor this PR after #363 is merged, for now I mostly care about you opinions regarding these changes.

### CODE REVIEW

**Comment by mrzzzrm** [comment]
should be ready @mrks

File: cmake/helpers.cmake [hunk 1]
```diff
@@ -0,0 +1,39 @@
+set(SANITIZER_COMPILE_FLAGS "-fsanitize=address,undefined -fno-sanitize-recover=all -fno-omit-frame-pointer")
+set(SANITIZER_LINK_FLAGS "-fsanitize=address")
```
Comment (mrks) [inline cmake/helpers.cmake]: Where did undefined go? :(

File: cmake/helpers.cmake [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+set(COVERAGE_COMPILE_FLAGS "-fprofile-arcs -ftest-coverage")
+
+macro(add_sanitizer_library arg1 arg2 arg3)
```
Comment (mrks) [inline cmake/helpers.cmake]: Is there a reason why we can't have real names for the args?

File: src/test/testing_assert.cpp [hunk 3]
```diff
@@ -0,0 +1,191 @@
+#include "testing_assert.hpp"
```
Comment (mrks) [inline src/test/testing_assert.cpp]: What is the advantage of having this in a separate file?

Comment (mrzzzrm) [reply src/test/testing_assert.cpp]: got ma reasons

**Comment by mrzzzrm** [comment]
@mrks CI said yes

**Pull request merged**